habs_portal/static/css/stations.css

```diff
@@ -10,6 +10,11 @@
   height: 100%;
 }
 
+.plotting-row {
+  height: 300px;
```

Comment:
Can the plot redraw on resize? I made all children of the plot panel height 100% but now you've fized them at 300?  I should actually try your PR before commenting, but this stood out.